### PR TITLE
Move screenChange to Display

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: smartmeter/display.cpp [hunk 1]
```diff
@@ -123,6 +127,23 @@ void Display::displayLogo()
...
+
+    counter++;
+    screenId = (int)counter / (targetFps * delayBetweenFrames);
+    screenId = screenId % numOfScreens;
+
+    if (screenId != lastScreenId) {
+        screens[screenId]();
+        lastScreenId = screenId;
+    }
+}
```
Comment (rafaellehmkuhl) [inline smartmeter/display.cpp:145]: Did you already test this block to see if the logic is working?

Everything else seens fine to me.

File: smartmeter/display.cpp [hunk 2]
```diff
@@ -123,6 +127,23 @@ void Display::displayLogo()
     display->display();
 }
 
+void Display::run()
+{
+    static uint counter = 0;
+    static uint screenId = 0;
+    static uint lastScreenId = -1;
```
Comment (Williangalvani) [inline smartmeter/display.cpp:134]: uint = -1?

File: smartmeter/display.cpp (hunk 1 above)
Comment (patrickelectric) [reply smartmeter/display.cpp:145]: Yep, here you can get the math.
```
K = n*16/(16*5)
id = K%3
id = (n/5)%3
```

File: smartmeter/display.cpp (hunk 2 above)
Comment (patrickelectric) [reply smartmeter/display.cpp:134]: This will be overwritten in the first integration.

File: smartmeter/display.cpp (hunk 1 above)
Comment (rafaellehmkuhl) [reply smartmeter/display.cpp:145]: Ok

**Pull request merged**